views/salesEmployee/create.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "templates/layout.njk" %}
```

Comment:
is this page screenreader friendly, just make sure lead components are aria checked

Reply:
yup they are